internal/common/resourceutil/cpu/cpu.go

```diff
@@ -182,7 +183,8 @@ func getCPUFreqCPUInfo() (float64, error) {
 
 	r := bufio.NewReader(f)
 	for {
-		line, err := r.ReadString('\n')
+		var line string
+		line, err = r.ReadString('\n')
```

Comment:
I don't know why the 'line' variable should be declared like this. Is there a problem with the previous code?
If necessary, what about declaring `line` in front of `for {`?

Reply:
In the previous code, when an error occurred in the `for { ... }`, the `err` variable did not return an error when exiting the function

Reply:
> If necessary, what about declaring `line` in front of `for {`?

Done! Thank you for advice!